client/my-sites/stats/stats-module/index.jsx

```diff
@@ -38,95 +40,92 @@ module.exports = React.createClass( {
 		this.setState( { noData: nextProps.dataList.isEmpty() } );
 	},
 
-	hasSummaryPage: function() {
-		var noSummaryPages = [ 'tags-categories', 'publicize' ];
-		return -1 === noSummaryPages.indexOf( this.props.path );
+	viewAll() {
```

Comment:
Something feels kind of funky here.  I don't know if we need this method but need to look a bit further.  If all we need to do is generate the URL we could do this logic in `getHref`